### PR TITLE
Telegraf

### PR DESCRIPTION
**Pull request by ezeev**
Telegraf installer which uses our packages (wavefront output plugin) hosted on packagecloud.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -879,38 +765,28 @@ EOF
...
+            FETCHER="wget --quiet -O /tmp/WF-CDPInstaller.tar.gz"
         elif command_exists curl; then
-            FETCHER="curl -L --silent -o /tmp/${APP_CONFIGURE_NAME}.tar.gz"
+            FETCHER="curl -L --silent -o /tmp/WF-CDPInstaller.tar.gz"
         else
             exit_with_failure "Either 'wget' or 'curl' are needed"
         fi
         echo_step "  Pulling application configuration file"
-        APP_LOCATION="https://github.com/kentwang929/install/files/464761/WF-PCInstaller.tar.gz"
+        APP_LOCATION="https://github.com/kentwang929/install/files/416326/WF-CDPInstaller.tar.gz"
```
Comment (jbau) [inline install.sh:775]: we probably need to repoint this at some point

File: install_collectd.sh [hunk 2]
```diff
@@ -0,0 +1,949 @@
...
+    if [ -n "$OVERWRITE_COLLECTD_CONFIG" ]; then
+        if command_exists wget; then
+            FETCHER="wget --quiet -O /tmp/collectd_conf.tar.gz"
+        elif command_exists curl; then
+            FETCHER="curl -L --silent -o /tmp/collectd_conf.tar.gz"
+        else
+            exit_with_failure "Either 'wget' or 'curl' are needed"
+        fi
+        echo_step "  Configuring collectd"
+        $FETCHER https://github.com/kentwang929/install/files/421074/default_collectd_conf.tar.gz >>${INSTALL_LOG} 2>&1
```
Comment (jbau) [inline install_collectd.sh:836]: this too

File: install_collectd.sh [hunk 3]
```diff
@@ -0,0 +1,949 @@
...
+    if [ "$APP_CONFIGURE" == "yes" ]; then
+        if command_exists wget; then
+            FETCHER="wget --quiet -O /tmp/${APP_CONFIGURE_NAME}.tar.gz"
+        elif command_exists curl; then
+            FETCHER="curl -L --silent -o /tmp/${APP_CONFIGURE_NAME}.tar.gz"
+        else
+            exit_with_failure "Either 'wget' or 'curl' are needed"
+        fi
+        echo_step "  Pulling application configuration file"
+        APP_LOCATION="https://github.com/kentwang929/install/files/464761/WF-PCInstaller.tar.gz"
```
Comment (jbau) [inline install_collectd.sh:889]: same

**Review by jbau — APPROVED** [review]
lgtm

File: install_collectd.sh (hunk 3 above)
Comment (ezeev) [reply install_collectd.sh:889]: Yeah... I disabled that part of the installer for now (it never gets called). I want to focus on getting Telegraf installer fully tested before attempting to re-organize an merge Chung's code.

**Pull request merged**